HANDLING_CODE_OF_CONDUCT_REPORTS.md

```diff
@@ -4,69 +4,40 @@ This document describes the proposed process for handling reports and escalation
 This process covers two types of reports based on the [Foundation's Code of Conduct Requirements](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md):
 
 * Reports for spaces managed by the Cross Project Council (CPC) which come in through `report@opensjsf.org`
-* Escalations which come in through `coc-escalation@openjsf.org`.   
+* Reports or escalations that have been brought to specific OpenJS Foundation projects and delegated to the CPC by that project’s leadership.   
```

Comment:
Does this item include:
* escalations when the reporter is not satisfied with the project's resolution
* escalations when the reporter wants to reach out to the foundation directly because it involves one of the project leads or members of their CoC Panel

I might be misunderstanding, but I'm not sure this description covers completely those scenarios or are those going to be handled in a different way?